example/mnist/mnist/evaluator.py

```diff
@@ -16,13 +16,16 @@
 
 class MNISTInference(PipelineHandler):
     def __init__(self) -> None:
-        super().__init__()
+        super().__init__(ppl_batch_size=5)
         self.device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
         self.model = self._load_model(self.device)
 
     @api(Image(shape=(28, 28, 3)), JsonOutput())
-    def ppl(self, img: Image, **kw: t.Any) -> t.Tuple[t.List[int], t.List[float]]:  # type: ignore
-        data_tensor = self._pre(img)
+    def single_ppl(self, img: Image, **kw: t.Any) -> t.Tuple[int, t.List[float]]:  # type: ignore
```

Comment:
Cloud we remove the single_ppl function?  `@api` maybe handle batch input and output automatically

Reply:
this depends on service implement,  the currently implement seems to only support single, is it? @jialeicui

Reply:
done